### PR TITLE
Refactor panels to better show whitespace

### LINKED ISSUE #552: single-pane static width
Currently the single-pane width is based on a responsive grid. Instead, lets try to make that box responsive but have a max-width.

### PR DESCRIPTION
**Pull request by mixonic**
Introduce a more whitespace-friendly pane layout. Closes #552 

This changes the layout DOM, and introduces new ways to size content:

* `.single-panel--wrapper.` a wrapper with a max-width of 40em (1em is 16px, so 640px). It also has a min-height to ensure it looks good even if there is not a lot of content.
* `.single-panel--wrapper.single-panel--wrapper--medium` a wrapper with a max-width of 45em or 720px.
* `.single-panel--wrapper.single-panel--wrapper--large` a wrapper with a max-width of 50em or 800px.
* `.single-panel--padded-content` is used inside `.single-panel--content`. It provides padding of 2em or 32px on all sides until the extra small breakpoint, and ensures the content is vertically centered.
* `.col-small-centered` is 66% of a panel's width until the extra small breakpoint.
* `.col-medium-centered` is 83% of a panel's width until the extra small breakpoint.

<details>
  <summary>Click to expand a long list of before/after screenshots</summary>
<br>
<strong>A) Login</strong>. Note that the content in the panel is horizontally and vertically centered. This is a <i>normal</i> size pane and has a max-width of 40em (1em is 16px, so 640px).
screen shot 2018-02-20 at 4 55 16 pm

**B) Login with error**. The box has a min-height (32em or 512px), and the content of the panel re-centers in the now shorter block.
screen shot 2018-02-20 at 4 56 03 pm

**C) Sign in success**. This is a *large* size pane and has a max-width of 50em or 800px. Again content is horizontally and vertically centered.
screen shot 2018-02-20 at 4 56 39 pm

**D) Enter name screen**. My window sizes here are 960px which is an awkward size, but this again is a large pane.
screen shot 2018-02-20 at 4 57 50 pm

**E) Enter name screen, zoomed out 75%**. This emulates a screen 1280px wide.
screen shot 2018-02-20 at 4 58 12 pm

**F) 2FA prompt**. A normal size panel, again with content horizontally and vertically centered.
screen shot 2018-02-20 at 4 58 57 pm

**G) 2FA Registrations**. A large panel. 
screen shot 2018-02-20 at 4 59 24 pm

**H) 2FA Registrations, zoomed to %75**. Because we're using max-width here instead of the grid system and breakpoints, the large panel basically stays the same size as the window gets bigger.
screen shot 2018-02-20 at 4 59 40 pm

**I) 2FA Registrations, mobile**. Fixed padding and margins.
screen shot 2018-02-20 at 5 00 32 pm

**J) TOTP Registration**. Again normal size panel.
screen shot 2018-02-20 at 5 01 12 pm

**K) TOTP Registration zoomed out**, again it doesn't change size at a larger window size.
screen shot 2018-02-20 at 5 01 36 pm

**L) TOTP Registration with error**
screen shot 2018-02-20 at 5 02 00 pm
</details>

### CODE REVIEW

**Comment by mixonic** [comment]
@jenn-rhim via DM sent a note about whitespace on **D** from above. There was some unexpected interaction between the class for the background image and the `col-small-centered` class. I've tweaked things and the screenshots follow:

<details>
   <summary>Click to expand screenshots</summary>

**1280 wide**
localhost_3000_publishers_email_verified

** 960 wide**
localhost_3000_publishers_email_verified 1

**425 wide**
localhost_3000_publishers_email_verified 2

**375 wide**
localhost_3000_publishers_email_verified 3

</details>

**Comment by ayumi** [comment]
a merge conflict from #614 i think

File: app/views/application/_icon_circled_check.html [hunk 1]
```diff
@@ -1,14 +1,6 @@
...
-        <g id="dialog" transform="translate(364.000000, 143.000000)">
-            <g id="icn-check" transform="translate(264.000000, 88.000000)">
-                <polygon id="Path-3" fill="#00BA99" points="13 35.2333261 17.106044 31.5025196 27.2748634 40.3526342 50.1751806 20 54 23.5034343 29.0434565 49 27.2748634 49"></polygon>
-                <circle id="Oval-4" stroke="#00BA99" stroke-width="5" cx="33" cy="33" r="33"></circle>
-            </g>
-        </g>
-    </g>
-</svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="72" height="72" viewBox="0 0 72 72">
```
Comment (mixonic) [inline app/views/application/_icon_circled_check.html]: @lgebhardt pinged me asking if this SVG file was in the right format. It was missing the correct formatting and I've fixed this in a separate commit of this PR.

@lgebhardt @jenn-rhim before exporting SVGs from Sketch you should install this compressor plugin: https://github.com/BohemianCoding/svgo-compressor not only does it save some bytes on the size, but additionally it removes the use of `id` attributes. When we have multiple SVGs on the page, the `id` attributes of uncompressed files can cause conflicts between the rendered images. Using this plugin is really a requirement of adding SVG to the site.

**Comment by mixonic** [comment]
@ayumi good to go!

**Pull request merged**